test/verify/test_conv_group_add.cpp

```diff
@@ -47,5 +47,5 @@ struct test_conv_group_add : verify_program<test_conv_group_add<DType>>
     std::string section() const { return "conv"; }
 };
 template struct test_conv_group_add<migraphx::shape::float_type>;
-// grouped  convolutions are not supported with MLIR therefore disable it
+// TODO grouped convolutions are not supported with MLIR therefore disable it
```

Comment:
Is this a TODO if so make this an issue to add support once MLIR supports group convolutions

Reply:
Not sure if MLIR plans to support group convolutions. Would need their support to handle fp8 gorup convolutions. https://github.com/ROCm/AMDMIGraphX/issues/3421

Reply:
That's fine. As long we can keep an eye on this and know this is a thing.